.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 
-rust: nightly
+rust: beta
```

Comment:
I think also AppVeyor will have to use beta? or are we going to ignore it for the backport?